egs2/mucs21_subtask1/asr1/local/test_data_prep.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
I notice `egs` and `egs2`'s preparation code are highly correlated. In that sense, please use symlink from one to another instead of copying all the code.